Feature: test
  Scenario: throw an error
    Given I throw an error
